src/main/java/jalau/at18/azul/WallBool.java

```diff
@@ -0,0 +1,25 @@
+package jalau.at18.azul;
+
+public class WallBool {
+    private final int wallBoolSize = 5;
+    private boolean[][] tileBool = new boolean[wallBoolSize][wallBoolSize];
+
+    public WallBool() {
+    }
+
+    public void intializeWallBoolean() {
+        for (int row = 0; row < wallBoolSize; row++) {
+            for (int column = 0; column < wallBoolSize; column++) {
+                tileBool[row][column] = false;
```

Comment:
This is a good idea, using booleans to know if tile has been placed, however, I think you could actually use objects to have one single wall. Something like:
```java
class Wall {
     private Tile[][] tilesPlaced;
}

class Tile {
      private boolean isPlaced
}
```

In that case you wouldn't need two walls, but one single wall with tiles that can tell you if they are placed or not

Reply:
@HugoS922 let me know any updates about this suggestion

Reply:
I changed the logic to use objects to have one single wall

Reply:
please first review the PR where you update the pyramid https://github.com/fjprog101/at18/pull/138/files